tools/draw_perfboard.py

```
@@ -0,0 +1,142 @@
+ROWY = {'P': 140, 'A': 285, 'B': 355, 'C': 425, 'D': 495, 'E': 565, 'G': 695}
+W, H = X0 + COLS * PITCH + 60, 1035
+
+def cx(c): return X0 + (c - 1) * PITCH
+def pos(c, row): return (cx(c), ROWY[row])
+
+img = Image.new("RGB", (W, H), "white")
+d = ImageDraw.Draw(img)
+
+def font(sz, bold=False):
+    for p in (("C:\\Windows\\Fonts\\arialbd.ttf" if bold else "C:\\Windows\\Fonts\\arial.ttf"),):
+        try: return ImageFont.truetype(p, sz)
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)